### DISCUSSION TITLE
Fresh install rules

### DISCUSSION THREAD

**Discussion by deannick** [Security-Onion-Solutions/securityonion#2877] [Unsupported Versions]
I have a fresh standalone install and everything seems to be working.  I am needing to silence some alerts, but I am not understanding the tuning on how to complete this task.  

One of the rules would be GPL SNMP public access udp.  I dont want to see this for my internal to internal ips, but keep it enabled for other alerts. Another one would be to completely disable the alert for ET POLICY Dropbox.com Offsite File Backup in Use.

**Comment by cm-ops** [comment]
The process for managing alerts is here https://docs.securityonion.net/en/latest/managing-alerts.html. On the GPL SNMP rule, you could modify the SID to !$HOME_NET. For the other you could disable the SID.

**Comment by rwwiv** [comment]
@deannick 
We're currently in the process of writing a tool to make disabling SIDs easier. If you'd like to test it out early, copy [this link's content](https://raw.githubusercontent.com/Security-Onion-Solutions/securityonion/feature/so-rules/salt/common/tools/sbin/so-rules) into `/usr/sbin/so-rules` on the manager (or alternatively, copy the script from the `feature/so-rules` branch of this repo - found under `securityonion/salt/common/tools/sbin`), then copy the following script somewhere onto the same system and run it as root:
``` sh
#!/bin/bash

. /usr/sbin/so-common

# Surround double curly braces in any minion pillar in single quotes so the pillars can be parsed as regular yaml

readarray -t minion_pillars <<< "$(find /opt/so/saltstack/local/pillar/minions -type f -name '*.sls')"
for pillar in "${minion_pillars[@]}"; do
  sed -i -r "s/ (\{\{.*}})$/ '\1'/g" "$pillar" 
done
```
---
After you're set up, just follow @cm-ops's advice on modifying/disabling those specific SIDs. The tool has built in help with `-h` to assist you with syntax.
